README.md

```diff
@@ -78,12 +78,27 @@ Here is a list and description of the different arguments to use for the script:
 
 <b>--url</b>
 * URL for desired shoe
+* Size parameter can also be passed in (for example: https://www.nike.com/launch/t/kobe-5-protro-bruce-lee?size=11). In this case, `--shoe-size` and `--shoe-type` will be ignored
```

Comment:
So is the advantage of using the `size` query param that it's less likely to fail since that size will already be pre-selected? Also wondering if `--shoe-type` can be expressed as a query param. I don't love the idea of allowing for both ways (especially when a combination of using both can cause issues). You could make it so that the `--shoe-size` param is appended to the `--url` instead of doing xpath to find the size button.

Reply:
Can only pass size as a query parameter. Wanted to give user flexibility to choose either option. My assumption was the query parameter would perform slightly faster than passing command line parameter and searching the DOM size selection section. (Not sure what Nike is doing exactly behind the scenes and I didn’t have time to run a performance analysis comparing each approach). As I noted in the ReadMe, passing in just the size as a query parameter on a release that has multiple shoes on he same page will result in an unintended shoe size & type selection.

Another observation is when users copy the url, the size query parameter is also there (I believe Nike appends it if the size is set in the user’s Nike profile).

So, options are: leave as is or use your approach, but we’ll need to do an additional check to see if the size-type command line parameter has been passed in.

Reply:
Alright, let's just leave it as is.